corehq/apps/reports_core/filters.py

```diff
@@ -137,6 +137,44 @@ def filter_context(self):
         }
 
 
+class NumericFilter(BaseFilter):
+    template = "reports_core/filters/numeric_filter.html"
+
+    def __init__(self, name, required=True, label='Numeric Filter', css_id=None):
```

Comment:
this should be marked for translation (either with `ugettext_lazy` or `ugettext_noop`)